src/test/java/com/stripe/BaseStripeTest.java

```diff
@@ -21,9 +21,11 @@
 import java.net.HttpURLConnection;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
+import java.util.Set;
```

Comment:
Nit, but can you tighten up the spacing here? I assume `java.util.Set` belongs with the group above instead of with `lombok.*`.

Reply:
Tiny follow along nit: I think what the original author was trying to do here was separate out different "groups" of imports. So all the `java` would land together, and then you'd have `com.*`, `org.*`, etc. in their own sections. If that's the case, I think the convention would be a double space between `java.*` and `lombok.*`.

Reply:
ah sorry @brandur-stripe: I simply used auto-formatting and assumed that was the standard one. I now fixed the import statement in my intellij setting to follow the order of the grouping here. With that though I didn't add double space between `java.*` and `lombok.*` as it the current one doesn't.